src/system_info/system_info_network_ivi.cc

```diff
@@ -24,12 +24,12 @@ void SysInfoNetworkTizen::StartListening() {
   vconf_notify_key_changed(VCONFKEY_NETWORK_STATUS,
       (vconf_callback_fn)OnNetworkTypeChanged, this);
 
-  if (!conn_ || !modem_path_)
+  if (!conn_)
```

Comment:
Need not check validation of `modem_path_`? In another word, is that legal that `modem_path_` is empty?